Tests/Runtime/MeshOps/DeleteElementsTests.cs

```diff
@@ -86,6 +86,65 @@ public static void DeleteRandomFace_CreatesValidMesh([ValueSource("shapeTypes")]
         }
     }
 
+    [Test]
+    public static void DeleteFirstFaceAndValidateMaterials([ValueSource("shapeTypes")] Type shape)
```

Comment:
Nice work adding a new test for this. It could be improved by testing more than one scenario. Consider what happens when two materials are removed? What if they are removed from the start, end, middle, or multiple interspersed?

Reply:
Right! :-) I've added more tests in [3772d72](https://github.com/Unity-Technologies/com.unity.probuilder/pull/460/commits/3772d72255c8887841c75887d34a1c1d4e104775)